src/client/rest/RESTMethods.js

```diff
@@ -377,6 +377,12 @@ class RESTMethods {
     return this.rest.makeRequest('get', Constants.Endpoints.channelMessage(channel.id, messageID), true);
   }
 
+  putGuildMember(guild, user, options) {
+    return this.rest.makeRequest('put', Constants.Endpoints.guildMember(guild.id, user.id), true, options).then(data =>
```

Comment:
Gus has a point, for consistency reasons it should stay as it is right now tho probably (looking at other methods).